core/src/executor/execute.rs

```diff
@@ -309,7 +312,27 @@ pub async fn execute<T, U: GStore<T> + GStoreMut<T>>(
 
             Ok((storage, Payload::Select { labels, rows }))
         }
+        Statement::ShowColumns { table_name } => {
+            let keys = try_block!(storage, {
+                let table_name = get_name(table_name)?;
+                let Schema { column_defs, .. } = storage
+                    .fetch_schema(table_name)
+                    .await?
+                    .ok_or_else(|| ExecuteError::TableNotFound(table_name.to_owned()))?;
```

Comment:
The current integration test `show columns` does not cover this error.
Let's add one for this :)